control-plane-operator/controllers/hostedcontrolplane/secretproviderclass/secretproviderclass.go

```diff
@@ -15,14 +15,15 @@ array:
   - |
     objectName: %s
     objectType: secret
+	objectEncoding: %s
```

Comment:
is the indentation okay here? Just want to be sure that this won't cause issue

Reply:
In my IDE, it's not showing up like that. I am having issues with testing this manually though.